Neos.Flow/Classes/Security/Authentication/Provider/PersistedUsernamePasswordProvider.php

```diff
@@ -100,10 +110,9 @@ public function authenticate(TokenInterface $authenticationToken)
             return;
         }
 
-        $authenticationToken->setAccount($account);
```

Comment:
^^If we do not set the account here, we wil not have it, if the validatePassword method doesn't return true.

Reply:
Right. But that's exactly the intention, no? => We don't want the token to contain the account if it could validate the password